src/pretalx/orga/forms/submission.py

```diff
@@ -196,7 +196,9 @@ class Meta:
         ]
         widgets = {
             "tags": forms.SelectMultiple(attrs={"class": "select2"}),
-            "track": forms.Select(attrs={"class": "select2"}),
+            "track": forms.Select(
```

Comment:
Have you tested the behaviour with mandatory track inputs? I assume the form won't submit while the field is empty, even with select2 shenanigans going on, so this will work out alright?